crates/sui-framework/sources/digest.move

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Sui types for message digests.
+module sui::digest {
```

Comment:
I thought that Move supports sha256 and sha3_256 in std can we crosscheck? @tnowacki 
Also can you sync up with @joyqvq who is working on a similar hash.move module?
See: https://github.com/MystenLabs/sui/pull/5149

Reply:
Right - https://github.com/MystenLabs/sui/blob/7424aba44aece90f2969171629773bdc6c79ed7e/crates/sui-framework/deps/move-stdlib/sources/hash.move#L9-L10 - I believe that we don't want to change existing interfaces, especially of Move stdlib. New APIs should probably use a similar (new) type.

Reply:
I can wait till this lands and add keccak to digest.move?